### PR TITLE
Polling with WinSocks on Windows

### PR DESCRIPTION
**Pull request by stevefan1999-personal**

### CODE REVIEW

File: examples/eventloop/poll.c [hunk 1]
```diff
@@ -6,7 +6,15 @@
 #include <errno.h>
 #include <string.h>
 #include <stdio.h>
+
+#ifdef _WIN32
```
Comment (svaarala) [inline examples/eventloop/poll.c]: There's a code style check for `#ifdef`s, so could you change to `#if defined(_WIN32)`? This should fix the Travis failure.

Comment (svaarala) [reply examples/eventloop/poll.c]: Hmm, maybe I'll just merge the change and do a fixup. I'll do that in a bit :)

**Comment by stevefan1999-personal** [comment]
😮So long it has been

**Comment by svaarala** [comment]
Better late than never, right? :)

**Pull request merged**